apps/tsilhqotin-language-hub/src/Components/Pages/Links/Links.tsx

```diff
@@ -1,4 +1,4 @@
-import { Divider } from '@mui/material';
+import { Card, Divider } from '@mui/material';
 import './Links.module.css';
 
 /* eslint-disable-next-line */
```

Comment:
We should have some local data that is an `ExternalLink[]` similar to the suggestion for `Funders`.